Imran Turganov

IMRUNya opened the pull request.

IMRUNya committed:
0 problem done

IMRUNya committed:
problem 1 solved

IMRUNya committed:
problem 2 partial

crassirostris commented:
## Problem 0

5/5

## Problem 1

Tests don't pass: https://gist.github.com/crassirostris/c139d477ded63653aca488e8b11f0ada

0/5

## Problem 2

Tests don't pass, creates a fork bomb instead

1/5

IMRUNya committed:
boost problem

IMRUNya commented:
> ## Problem 0
> 5/5
> 
> ## Problem 1
> Tests don't pass: https://gist.github.com/crassirostris/c139d477ded63653aca488e8b11f0ada
> 
> 0/5
> 
> ## Problem 2
> Tests don't pass, creates a fork bomb instead
> 
> 1/5

Could you please check again Problem 1, it was working, but at the end I don't no why I added boost. Now I deleted it and it works in my computer.

crassirostris commented:
OK, problem 1 5/5
